src/transformState.js

```diff
@@ -5,7 +5,29 @@
  * @param {Object[]} actions
  */
 function transformState(state, actions) {
-  // write code here
+  for (const obj of actions) {
+    switch (obj.type) {
+      case 'addProperties':
+        Object.assign(state, obj.extraData);
+
+        break;
+
+      case 'removeProperties':
+        for (const removeKey of obj.keysToRemove) {
+          delete state[removeKey];
+        }
```

Comment:
![image](https://github.com/mate-academy/js_stateful-object/assets/139622699/50a6bbbc-7c28-46f8-bf42-f78b31c82c2a)
Better use forEach()

Reply:
Not fixed.